src/main/java/com/palantir/websecurity/CorsConfiguration.java

```diff
@@ -59,13 +59,15 @@ public boolean enabled() {
     public abstract Optional<Long> preflightMaxAge();
 
     @ValidationMethod(message = "preflightMaxAge can't be negative")
+    @SuppressWarnings("unused")
```

Comment:
These methods (isPreflightMaxAgeNegative, isAllowedOriginsValid) are never explicitly called which caused errorprone warnings, in my understanding these are implicitly called by dropwizard. https://www.dropwizard.io/en/stable/manual/validation.html#annotations
We should supress these warnings as we know that these methods are not problematic.